master/pkg/model/model.go

```diff
@@ -6,36 +6,21 @@ import (
 	"time"
 )
 
-// Model represents a row from the `models` table.
+// Model represents a row from the `models` table. Unused except for tests.
 type Model struct {
-	ID              int       `db:"id" json:"id"`
 	Name            string    `db:"name" json:"name"`
 	Description     string    `db:"description" json:"description"`
-	Metadata        JSONObj   `db:"metadata" json:"metadata"`
 	CreationTime    time.Time `db:"creation_time" json:"creation_time"`
 	LastUpdatedTime time.Time `db:"last_updated_time" json:"last_updated_time"`
+	Metadata        JSONObj   `db:"metadata" json:"metadata"`
+	ID              int       `db:"id" json:"id"`
 	Labels          []string  `db:"labels" json:"labels"`
-	Username        string    `db:"username" json:"username"`
+	UserID          int       `db:"user_id" json:"user_id"`
```

Comment:
```suggestion
	UserID          UserID       `db:"user_id" json:"user_id"`
```
we have a userID type of ```model.UserID```